test/js2r-logging-test.el

```diff
@@ -0,0 +1,86 @@
+;;; js2r-formatting-test.el --- Unit tests for js2r-formatting.el  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2017 Felipe Ochoa
+
+;; Author: Felipe Ochoa <felipe@fov.space>
+;; Keywords: test
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+;;; Code:
+
+(require 'js2-refactor)
+(require 'buttercup)
```

Comment:
I added buttercup tests because I wanted to test a function before refactoring it, but ecukes wasn't the proper tool for that.

Unless you want to add unit tests for a function in isolation (which doesn't seem to be your case), I'd go with ecukes tests only.